agent/config/runtime_test.go

```diff
@@ -286,14 +286,14 @@ func TestConfigFlagsAndEdgecases(t *testing.T) {
 				rt.ConsulRaftElectionTimeout = 52 * time.Millisecond
 				rt.ConsulRaftHeartbeatTimeout = 35 * time.Millisecond
 				rt.ConsulRaftLeaderLeaseTimeout = 20 * time.Millisecond
-				rt.ConsulSerfLANGossipInterval = 100 * time.Millisecond
-				rt.ConsulSerfLANProbeInterval = 100 * time.Millisecond
-				rt.ConsulSerfLANProbeTimeout = 100 * time.Millisecond
-				rt.ConsulSerfLANSuspicionMult = 3
-				rt.ConsulSerfWANGossipInterval = 100 * time.Millisecond
-				rt.ConsulSerfWANProbeInterval = 100 * time.Millisecond
-				rt.ConsulSerfWANProbeTimeout = 100 * time.Millisecond
-				rt.ConsulSerfWANSuspicionMult = 3
+				rt.GossipLANGossipInterval = 100 * time.Millisecond
+				rt.GossipLANProbeInterval = 100 * time.Millisecond
+				rt.GossipLANProbeTimeout = 100 * time.Millisecond
+				rt.GossipLANSuspicionMult = 3
+				rt.GossipWANGossipInterval = 100 * time.Millisecond
+				rt.GossipWANProbeInterval = 100 * time.Millisecond
+				rt.GossipWANProbeTimeout = 100 * time.Millisecond
+				rt.GossipWANSuspicionMult = 3
```

Comment:
Why don't we have tests for `Gossip*Nodes` and `Gossip*RetransmitMult` here again?

Reply:
That test is just for -dev mode overrides. I guess we could through validation for some other values not being overridden but other tests already cover that.